search/api.py

```diff
@@ -123,10 +184,27 @@ def _apply_general_query_filters(search, user):
 
     course_filter = Q("term", published=True) | ~Q("terms", object_type=[COURSE_TYPE])
 
+    bootcamp_filter = Q("term", published=True) | ~Q(
+        "terms", object_type=[BOOTCAMP_TYPE]
+    )
+
+    program_filter = ~Q("terms", object_type=[PROGRAM_TYPE])
```

Comment:
How should we query for Programs?

Reply:
For now, we'll only query the Program metadata, i.e. program description and program title. We won't query across courses that are _in_ the program list.